app/serializers/application.js

```diff
@@ -5,14 +5,11 @@ export default DS.JSONAPISerializer.extend(DataTableSerializerMixin, {
 
   // eslint-disable-next-line no-unused-vars
   shouldSerializeHasMany(snapshot, key, relationshipType) {
-    const shouldSerialize = this._super(...arguments);
-    const serializeOption = relationshipType.options || {
-      serialize: true,
-    };
-    if (typeof serializeOption.serialize !== 'undefined') {
-      return shouldSerialize && serializeOption.serialize;
+    // If serialization option specified, use that. Otherwise use Ember Data defaults
+    if (relationshipType.options && (typeof relationshipType.options.serialize !== 'undefined')) {
+      return relationshipType.options.serialize;
     }
-    return shouldSerialize;
+    return this._super(...arguments);
```

Comment:
sneaky :smirk: 
Dit komt van https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/683/files
Maar die PR is nog niet approved.
Ik zal bij testen van deze feature rekening houden met deze change, en kijken of ik iets kapot krijg door deze change (want er zijn nog geen andere changes in de models gebeurt hiervoor)

Reply:
PR #683 has been merged